content/learn/07.built-in-libraries/04.software-serial/software-serial.md

```diff
@@ -21,13 +21,24 @@ SoftwareSerial library has the following known limitations:
 
 * It cannot transmit and receive data at the same time.
 * If using multiple software serial ports, only one can receive data at a time.
-* Not all pins on the Mega and Mega 2560 boards support change interrupts, so only the following can be used for RX: 10, 11, 12, 13, 14, 15, 50, 51, 52, 53, A8 (62), A9 (63), A10 (64), A11 (65), A12 (66), A13 (67), A14 (68), A15 (69).
-Not all pins on the Leonardo and Micro boards support change interrupts, so only the following can be used for RX: 8, 9, 10, 11, 14 (MISO), 15 (SCK), 16 (MOSI).
-* On Arduino or Genuino 101 boards the current maximum RX speed is 57600bps. 
-* On Arduino or Genuino 101 boards RX doesn't work on digital pin 13.
+* On some boards, not all pins can be used. See the "**Supported Pins**" table below for details.
+* On Arduino or Genuino 101 boards the current maximum RX speed is 57600bps.
 
 If your project requires simultaneous data flows, see Paul Stoffregen's [AltSoftSerial library](https://www.pjrc.com/teensy/td_libs_AltSoftSerial.html).
 
+### Supported Pins
+
+| Board                                                                                               | RX                                           | TX        |
+| --------------------------------------------------------------------------------------------------- | -------------------------------------------- | --------- |
+| [Arduino 101](/retired/boards/arduino-101-619/)                                                     | 0-12, 14 (A0) - 19 (A5)                      |           |
```

Comment:
@per1234 since we are not mentioning "Arduino" in the rest of the boards, do you think it would be enough to say "101" here?

Reply:
I think it is a good idea. I think there isn't much danger of the reader being confused since the board name links to documentation path for the 101.

I have now made the suggested change:

https://github.com/arduino/docs-content/compare/c3eb74b1c56f65f744cee2e5a1b7fe9412cf383d..7b88e8630795f2c24bbae4d2531eb813cef502f3